### DISCUSSION TITLE
Extended GFM Tables package!

### DISCUSSION THREAD

**Discussion by calculuschild** [markedjs/marked#2321] [Show and tell]
I put together a Marked extension package that allows table cells to span rows and columns. I don't have any experience publishing my own NPM packages before now but it seems to work...?

I'm not looking for feedback on the extension itself necessarily (although that is welcome), but I would appreciate it if there's any NPM-specific structure I need to correct to make this package "work smoothly" with Marked for any users.

https://www.npmjs.com/package/marked-extended-tables

**Comment by calculuschild** [comment]
Thanks for taking a look at it @UziTech ! That was a huge help!

**Reply by UziTech** [reply]
It is great to see these extensions being used! Thanks for creating that.

Maybe we should have a list of known extensions somewhere in the docs?

**Reply by calculuschild** [reply]
I was thinking the same thing.
